src/ui-scroll.js

```diff
@@ -559,18 +560,31 @@ angular.module('ui.scroll', [])
 
           adapter.reload = reload;
 
+          var scrolling = false;
+
           // events and bindings
           viewport.bind('resize', resizeAndScrollHandler);
-          viewport.bind('scroll', resizeAndScrollHandler);
+          viewport.bind('scroll', resizeHandler);
```

Comment:
I think `resizeHandler ` should be renamed to `scrollHandler`, and `resizeAndScrollHandler ` to `resizeHandler `